.bit/responses/3.2-Week 3 Step 2.md

```diff
@@ -1,109 +1,75 @@
 
-async function create(client, databaseId, containerId) 
-{
-   const { database } = await client.databases.createIfNotExists({
-      id: config.databaseId
-   });
+<details>
+<summary>:exclamation: Can I get a more detailed list of steps?</summary>
+  </br>
+  
+1. In the **async function**, create a new constant that will recieve the dominant emotion from your emotion data (i.e. angry, happy). This dominant emotion should be determined before you call on the trigger function.
+2. Using **node-fetch**, call the Giphy **translate endpoint** from the documentation.
+3. Run the constant through the **string** parameter in the endpoint (look as the documentation if this sounds confusing). Add your **Giphy API key**, and set the number of returned gifs to 1 by adding `&limit=1` to the endpoint.
+4. Declare another constant that formats the results of the API call in **json** format.
+5. Make this second constant the **output** of the function in `context.res`.
 
-   const { container } = await client
-      .database(config.databaseId)
-      .containers.createIfNotExists(
-         { id: config.containerId, config.partitionKey },
-         { offerThroughput: 400 }
-      );
-   }
-}
+  <br><br/>
+  
+</details>
 
-export default create;
+<details>
+<summary>:exclamation: I'm stuck. Can I get some hints?</summary>
+  </br>
+  
+Fill in the gaps:
+  
 ```
-
-
-
-## **:pencil: Task 2: Set up your Azure Cosmos DB configuration by creating the `config.js` and `databaseContext.js` files!**
-
-
-## **:camping: Make sure to commit ONLY `databaseContext.js` (NOT `config.js`) to the root directory of a new branch `week3` in this repository to move on!**
-
-Remember that you should NEVER be committing and pushing your subscription key into GitHub. Leave your `config.js` file out of the push. See this [link](https://docs.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository) to see how to remove unwanted files from a repo's history.
\ No newline at end of file
+module.exports = async function (context, req) {
+    context.log(‘JavaScript HTTP trigger function processed a request.‘);
+    const phone = (req.query.phone || (req.body && req.body.phone));
+    
+    const emotion = (FILL_THIS_IN);
+    
+    const apiResult = await fetch (“ENDPOINT_GOES_HERE");
+    const jsonResult = await ADD_CODE.json();
+    
+    context.res = {
+        // status: 200, /* Defaults to 200 */
+        body: jsonResult.data.images.downsized_large.url
+    };
+ ```
+
+  <br><br/>
+  
+</details>
+
+_Don't forget to have fun with the API – customize your call using the different parameters. Remember to use the gif and **not** the sticker endpoint!_
+
+## **:camping: To move on, commit your updated trigger function code as a `.js` file to the root directory of the `week3` branch.**
+
+*Remember: **remove** your Giphy key and Twilio keys before submitting!*

```

Comment:
Do you know how to add environment secrets? this would be best for the students so then they don't worry about leaving keys in the code (and is better industry practice)